setup.py

```diff
@@ -193,8 +193,9 @@ def run(self):
         'pillow>=8.3.2',
         'pygobject>=3.40.1',
         'sqlalchemy>=1.4.36,<2',
-        'toml>=0.10.2',
-        'recipe-scrapers>=14.27.0,<15',
+        'tomli_w>=1.0.0',
+        'toml==0.10.2; python_version<"3.11"',
+        'recipe-scrapers>=14.27.0',
```

Comment:
This change is unrelated to this PR.

```suggestion
        'recipe-scrapers>=14.27.0,<15',
```